adapters/tlsio_openssl.c

```diff
@@ -880,13 +1313,52 @@ static int load_system_store(TLS_IO_INSTANCE* tls_io_instance)
             X509_free(x509);
         }
     }    
+
+    // load all the revocation lists into the openSSL crl store
+    PCCRL_CONTEXT pCRLContext = NULL;
+    while (1)
+    {
+        /* To free a context obtained by a find or enumerate function,
+         * either pass it in as the previous context parameter to a
+         * subsequent invocation of the function, or call the appropriate
+         * free function. --from MSDN
+        */
+        pCRLContext = CertEnumCRLsInStore(hSysStore, pCRLContext);
```

Comment:
Do you need to free the last PCRLContext?

Reply:
there is a "trick", doc says, the passed in context is freed.